quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/Picocli.java

```diff
@@ -788,7 +788,8 @@ private static String getDecoratedOptionDescription(PropertyMapper<?> mapper) {
             }).toList();
 
             var isStrictExpectedValues = mapper.getOption().isStrictExpectedValues();
-            var printableValues = String.join(", ", decoratedExpectedValues) + (!isStrictExpectedValues ? ", or a custom one" : "");
+            var isCaseSensitiveExpectedValues = mapper.getOption().isCaseInsensitiveExpectedValues();
+            var printableValues = String.join(", ", decoratedExpectedValues) + (!isStrictExpectedValues ? ", or a custom one" : (isCaseSensitiveExpectedValues ? " (case insensitive)" : ""));
```

Comment:
IMO, case sensitivity should not be associated with the strictness of expected values. It means when we allow specifying some custom values, we can also say whether we accept case-insensitive values.

I would rather add the info about case sensitivity before the list of expected values. See a suggestion below.

cc @shawkins